src/lib/Routes.ts

```diff
@@ -0,0 +1,66 @@
+export enum Routes {
+    Dashboard = "/dashboard",
+    Notification = "/notification",
+    Attendance = "/attendance",
+    MarkAttendance = "/mark-attendance",
+    Fund = "/fund",
+    Member = "/member",
+    AddMember = "/add-member",
+    ShiftIncharge = "/shift-incharge",
+    AddShiftIncharge = "/add-shift-incharge",
+    MissingShoes = "/missing-shoes",
+    AddMissingShoes = "/add-missing-shoes",
+    LeaveRequest = "/leave-request",
+    AddLeaveRequest = "/add-leave-request",
+}
+
+export const RoutesTitle = [
```

Comment:
Why do we need this?

Reply:
We have a common header, so I've added titles that correspond to the matching routes.